### DISCUSSION TITLE
Is it able to adjust launcher UI longer or bigger?

### DISCUSSION THREAD

**Discussion by helsmy** [Ulauncher/Ulauncher#1334] [Q&A] [answered]
My display zoom is set to 100% and the length of launcher UI is less than 1/3 of the screen. It is a little bit too short for me. Is there a way to make UI longer or bigger?  I did not find settings about it on preference.

**Comment by friday** [comment]
There is no setting for this yet.

**Comment by friday** [comment, marked as answer]
Update: this was added in Ulauncher [v6.0.0-beta6](https://github.com/Ulauncher/Ulauncher/releases/tag/v6.0.0-beta6)
